src/IndexService/ProductList/ElasticSearch/AbstractElasticSearch.php

```diff
@@ -28,7 +28,7 @@
 
 abstract class AbstractElasticSearch implements ProductListInterface
 {
-    const LIMIT_UNLIMITED = 'unlimited';
+    const LIMIT_UNLIMITED = -1;
```

Comment:
Please also change the PHPDoc below

https://github.com/pimcore/ecommerce-framework-bundle/blob/7ca7952c068c83be62ed42457d561c3cdb112745/src/IndexService/ProductList/ElasticSearch/AbstractElasticSearch.php#L324